pwndbg/commands/version.py

```diff
@@ -144,8 +146,19 @@ def bugreport(run_browser=False, use_gh=False):
 
     gdb_config = gdb.execute("show configuration", to_string=True).split("\n")
     all_info = all_versions()
+    os_info = system()
 
     current_setup = "Platform: %s\n" % platform()
+
+    if os_info.lower() == "linux" and os.path.isfile("/etc/os-release"):
+
+        with open("/etc/os-release", "r") as os_release:
+            contents = os_release.read()
+            match = re.search(r"PRETTY_NAME=(.+)", contents)
```

Comment:
Why don't we match against `PRETTY_NAME="(.+)"` so we can avoid the `replace` call below? Or is it not guaranteed that there will be quotes here?

Reply:
I’m unsure if the quotes are guaranteed however when I get home I  can change the regex to:

PRETTY_NAME=“?(.+)”? 

Or something like that to get rid of the replace call below.